WebApp/src/main/java/ev/projects/webapp/services/DocumentService.java

```diff
@@ -0,0 +1,45 @@
+package ev.projects.webapp.services;
+
+import ev.projects.Document;
+import ev.projects.webapp.repositories.IDocumentRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.io.InputStream;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+@Service
+public class DocumentService implements IDocumentService {
+
+    private IDocumentRepository documentRepository;
+
+    @Autowired
+    public DocumentService(IDocumentRepository documentRepository) {
+        this.documentRepository = documentRepository;
+    }
+
+
+    @Override
+    public List<Document> getAllDocumentsByCase(long caseID) {
+        return documentRepository.getAllDocumentsByCase(caseID);
+    }
+
+    @Override
+    public List<Document> getAllAttachmentsByDocument(long documentID) {
+        return documentRepository.getAllAttachmentsByDocument(documentID);
+    }
+
+    @Override
+    public byte[] downloadDocument(long ID) throws Exception {
```

Comment:
The name of the method is not correct. The service should not know the purpose to retrieve the doc file.
Better name would be getDocumentContent or getDocumentFile

Reply:
Changed in #9313b18